spec/features/articles_edited_spec.rb

```diff
@@ -1,23 +1,132 @@
 # frozen_string_literal: true
-
 require 'rails_helper'
 
+slug = 'foo/bar'
+
 describe 'Articles Edited view', type: :feature, js: true do
-  let(:course) { create(:course, start: '2017-01-01', end: '2020-01-01') }
   let(:user) { create(:user, username: 'Ragesoss') }
-  let(:article) { create(:article, title: 'Nancy_Tuana') }
-
+  let(:home_wiki) { Wiki.get_or_create language: 'en', project: 'wikipedia' }
+  let(:es_wiktionary) { create(:wiki, language: 'es', project: 'wiktionary') }
+  let(:fr_wikidata) { create(:wiki, language: 'fr', project: 'wikidata') }
+  
   before do
+    stub_wiki_validation
+    course = create(:course,
+      slug: slug,
+      start: '2017-01-01',
+      end: '2021-01-01',
+      home_wiki_id: home_wiki.id
```

Comment:
In most cases, you can and should be a little more concise in the tests by passing the object instead of the idea for `create`. In this case, `home_wiki: home_wiki`. The same goes for the `wiki` of each article below.

Reply:
Thanks! Have changed this